test/runtests.jl

```diff
@@ -1,13 +1,14 @@
 using NgSpice
 using Test
+using SafeTestsets
 
-rootdir = joinpath(@__DIR__, "..")
-netpath = joinpath(rootdir, "inputs", "mosfet.cir")
-
-@testset "interface functions" begin
+@safetestset "interface functions" begin
     include("ngspice_api.jl")
 end
-
 @testset "friend functions" begin
     include("wrapper_api.jl")
 end
+@safetestset "high level functions" begin
+    include("ngspice_high_level.jl")
```

Comment:
I've used `ngspice` prefix to refer the first level interface functions and `wrapper` prefix to the Julia-user-centric functions. So I'll rename this as `wrapper_safetests.jl`